_python/wiki_scrapping.py

```diff
@@ -0,0 +1,103 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+import numpy as np
+from sqlalchemy import create_engine
+import numpy as np
+import os
+import logging
+
+
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s',level=logging.INFO)
+
+class Wiki_ParseWar:
+    #this is an instance method
+    def __init__(self,url):
+        self.url = url
+        logging.info('Database connetion was establish successfully')
```

Comment:
I think this line is lying to us as it is simply not true. You can make it as 'WikiParser instance is created'